server/depot.go

```diff
@@ -98,14 +112,93 @@ func (d fileDepot) Serial() (*big.Int, error) {
 		}
 		return s, nil
 	}
-	data, err := ioutil.ReadFile(name)
+	file, err := os.Open(name)
 	if err != nil {
 		return nil, err
 	}
-	serial := s.SetBytes(data)
+	defer file.Close()
+	r := bufio.NewReader(file)
+	// is there a better way??
+	data, err := r.ReadString('\r')
+	data = strings.TrimSuffix(data ,"\r")
+	data = strings.TrimSuffix(data ,"\n")
+	if err != nil && err != io.EOF {
+		return nil, err
+	}
+	serial, success := s.SetString(data,16)
+	if success != true  {
+		return nil, errors.New("Could not convert "+string(data)+" to serial number")
```

Comment:
error strings should not be capitalized 
https://github.com/golang/go/wiki/CodeReviewComments#error-strings